librz/bin/format/elf/elf_dynamic.h

```diff
@@ -0,0 +1,17 @@
+// SPDX-FileCopyrightText: 2021 08A <08A@riseup.net>
+// SPDX-FileCopyrightText: 2021 RizinOrg <info@rizin.re>
+// SPDX-License-Identifier: LPGL-3.0-only
+
+#include <ht_uu.h>
+#include "elf.h"
+
+#ifndef _INCLUDE_ELF_DYNAMIC_H_
+#define _INCLUDE_ELF_DYNAMIC_H_
+
+typedef struct Elf_(rz_bin_elf_dt_dynamic) {
+	HtUU *info;
+	RzVector *dt_needed;
+}
+RzBinElfDtDynamic;
```

Comment:
```suggestion
} RzBinElfDtDynamic;
```

Reply:
It was clang format. But I remove the Elf_() macro (unused) and format again.